### DISCUSSION TITLE
Boomerang Thailand

### DISCUSSION THREAD

**Discussion by omaromarver** [iptv-org/community#785] [Show and tell]
### Channel Name

Boomerang Thailand

### Channel ID

Boomerang.th

### Website

https://www.boomerangthailand.tv/#/app/home

### Stream URL

http://d8380c4177d6.sn.mynetname.net:25461/demo/demo/25

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Xtream-Codes server. We do not accept these kind of links.
![image](https://user-images.githubusercontent.com/30985701/167222268-ed420bd1-4fe6-4db4-ae48-b86c409ff85e.png)

**Comment by freearhey** [comment]
The channel has been added to our blocklist due to the claims of the copyright holder: [#16839](https://github.com/iptv-org/iptv/issues/16839)
